styles/signals.mapcss

```diff
@@ -730,13 +737,11 @@ node|z16-[railway=signal]["railway:signal:direction"]["railway:signal:train_prot
 	icon-height: 20;
 	allow-overlap: true;
 }
-node|z16-17[railway=signal]["railway:signal:direction"]["railway:signal:train_protection"="DE-ESO:blockkennzeichen"]::blockkennzeichen
+node|z16-[railway=signal]["railway:signal:direction"]["railway:signal:train_protection"="DE-ESO:blockkennzeichen"]::blockkennzeichen
 {
 	z-index: 8510;
 	text: "ref";
 	text-offset-x: -28;
```

Comment:
`text-offset-x` should be set to `-10` to align the text in the middle of the sign in JOSM.